apps/settings/serializers/feature.py

```diff
@@ -138,7 +140,23 @@ class ChatAISettingSerializer(serializers.Serializer):
         help_text=_('The proxy server address of the GPT service. For example: http://ip:port')
     )
     GPT_MODEL = serializers.ChoiceField(
-        default=GPT_MODEL_DEFAULT, choices=GPT_MODEL_CHOICES, label=_("GPT Model"), required=False,
+        default=GPTModelChoices.gpt_4o_mini, choices=GPTModelChoices.choices,
+        label=_("GPT Model"), required=False,
+    )
+    DEEPSEEK_BASE_URL = serializers.CharField(
+        allow_blank=True, required=False, label=_('Base URL'),
+        help_text=_('The base URL of the Chat service.')
+    )
+    DEEPSEEK_API_KEY = EncryptedField(
+        allow_blank=True, required=False, label=_('API Key'),
+    )
+    DEEPSEEK_PROXY = serializers.CharField(
+        allow_blank=True, required=False, label=_('Proxy'),
+        help_text=_('The proxy server address of the GPT service. For example: http://ip:port')
+    )
+    DEEPSEEK_MODEL = serializers.ChoiceField(
+        default=DeepSeekModelChoices.deepseek_chat, choices=DeepSeekModelChoices.choices,
+        label=_("DeepSeek Model"), required=False,
     )
 
 
```

Comment:
I see a minor difference in your changes:

1️⃣ There is an unnecessary comma before "date_expired_default" in `datetime_now` at line 22.
  
However, there doesn't appear to be anything significant that warrants major modifications. Your code looks good and should work fine.

As far as optimizations go, you might consider:

- Refactoring methods (e.g., use `@classmethod` instead of manually setting the attribute).
- Consider if using classes can simplify structure.

Also remember that while these small improvements don't impact functionality, considering efficiency could further improve performance under specific conditions or load requirements.

Feel free to ask for more detailed advice about Django or any other topic!